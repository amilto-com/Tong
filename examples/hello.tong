// Hello World in TONG
// Demonstrates basic syntax and output

fn main() {
    print("Hello, TONG World!")
    print("The ultimate programming language is here!")
    
    // Variables with type inference
    let name = "TONG"
    let version = 1.0
    let is_awesome = true
    
    print("Language:", name)
    print("Version:", version) 
    print("Awesome:", is_awesome)
}

main()