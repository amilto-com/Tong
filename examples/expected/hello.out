Hello, TONG World!
The ultimate programming language is here!
Language: TONG
Version: 1.0
Awesome: true
